utilities-core/src/test/java/uk/gov/justice/services/common/converter/ObjectMapperProducerTest.java

```diff
@@ -275,7 +285,8 @@ public String toString() {
         private final int age;
         private final Colour favouriteColour;
 
-        public Person(final String name, final int age, final Colour favouriteColour) {
+        @JsonCreator
+        public Person(@JsonProperty("name") final String name, @JsonProperty("age") final int age, @JsonProperty("favouriteColour") final Colour favouriteColour) {
```

Comment:
We shouldn't need these annotations - in fact, adding them means that we are not testing that the object mapper works without them. If you're finding the tests fail withoout them when running from your IDE, ist probably because you haven't got the "-parameters" compiler option turned on.